estate/models/fields/estate_property.xml

```diff
@@ -0,0 +1,204 @@
+        <field name="name">x_postcode</field>
+        <field name="field_description">Postcode</field>
+        <field name="ttype">char</field>
+    </record>
+
+     <!-- Fields From Exercise -->
+     <record id="field_real_estate_date_availability" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_date_availability</field>
+        <field name="field_description">Date Availability</field>
+        <field name="ttype">date</field>
+    </record>
+
+    <record id="field_real_estate_expected_price" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_expected_price</field>
+        <field name="field_description">Expected Price</field>
+        <field name="ttype">float</field>
+        <field name="required">True</field>
+    </record>
+
+    <record id="field_real_estate_bedrooms" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_bedrooms</field>
+        <field name="field_description">Bedrooms</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_living_area" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_living_area</field>
+        <field name="field_description">Living Area</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_facades" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_facades</field>
+        <field name="field_description">Facades</field>
+        <field name="ttype">integer</field>
+    </record>
+
+    <record id="field_real_estate_garage" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_garage</field>
+        <field name="field_description">Garage</field>
+        <field name="ttype">boolean</field>
+    </record>
+
+    <record id="field_real_estate_garden" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_garden</field>
+        <field name="field_description">Garden</field>
+        <field name="ttype">boolean</field>
+    </record>
+
+    <record id="field_real_estate_garden_area" model="ir.model.fields">
+        <field name="model_id" ref="estate.model_real_estate_property"/>
+        <field name="name">x_garden_area</field>
+        <field name="field_description">Garden Area</field>
+        <field name="ttype">integer</field>
+        <field name="depends">x_garden</field>
+        <field name="compute"><![CDATA[for property in self:
+            property['x_garden_area'] = 0 if not property.x_garden else 1000
```

Comment:
back to this: this is not correct. It is in an onchange, to set back to a default value when editing the boolean in the form. But here you would totally erase the value entered by the user.